### PR TITLE
More detailed data backup explanation

### PR DESCRIPTION
**Pull request by TheLastProject**

### CODE REVIEW

File: _pages/faq.md [hunk 1]
```diff
@@ -7,11 +7,25 @@ include_in_footer: true
...
+
+No. Catima does not collect any data and is designed to work fully offline. See also our [Privacy Policy](/privacy-policy).
+
+### Can I back up and restore my data?
+
+Catima supports both manual backups (through Catima's built-in Import/Export functionality) and automatic backups through Android Auto Backup.
+
+To back up or restore Catima's data manually, click the ⋮ (menu) button and choose Import/Export. Here you can export your data into a (password protected if wanted) .zip file and later restore it.
+
+When Android Auto Backup is enabled (it is enabled by default on most Android systems) [Google will automatically make encrypted backups of apps on their servers](https://support.google.com/android/answer/2819582). On custom ROMs like LineageOS, CalyxOS, etc., backups are generally handled by [SeedVault](https://calyxinstitute.org/projects/seedvault-encrypted-backup-for-android) instead. However, for reliability reasons, it is recommended to use Catima's built-in Import/Export functionality instead as this functionality is better tested.
```
Comment (obfusk) [inline _pages/faq.md:22]:
```suggestion
When Android Auto Backup is enabled (which it is by default on most Android systems) [Google will automatically make encrypted backups of apps on their servers](https://support.google.com/android/answer/2819582). On custom ROMs like LineageOS, CalyxOS, etc., backups are generally handled by [SeedVault](https://calyxinstitute.org/projects/seedvault-encrypted-backup-for-android) instead. However, for reliability reasons, it is recommended to use Catima's built-in Import/Export functionality instead as this functionality is better tested.
```

File: _pages/faq.md [hunk 2]
```diff
@@ -7,11 +7,25 @@ include_in_footer: true
...
+
+### Does Catima collect any data?
+
+No. Catima does not collect any data and is designed to work fully offline. See also our [Privacy Policy](/privacy-policy).
+
+### Can I back up and restore my data?
+
+Catima supports both manual backups (through Catima's built-in Import/Export functionality) and automatic backups through Android Auto Backup.
+
+To back up or restore Catima's data manually, click the ⋮ (menu) button and choose Import/Export. Here you can export your data into a (password protected if wanted) .zip file and later restore it.
```
Comment (obfusk) [inline _pages/faq.md:20]:
```suggestion
To back up or restore Catima's data manually, click the ⋮ (menu) button and choose Import/Export. Here you can export your data into a .zip file and later restore it. The .zip file can be protected with a password for extra security.
```

Also: might look nicer to wrap `⋮` in a `<kbd>` tag or something?

Comment (TheLastProject) [reply _pages/faq.md:20]: No strong opinion, which do you prefer?

With:
image

Without:
image

Feel free to edit your suggestion to include <kbd> if you prefer that version.

Comment (obfusk) [reply _pages/faq.md:20]: I prefer the `<kbd>`. I use that in the help for Jiten myself (though it's styled a bit differently by my CSS):

image

Could always change the CSS as well if you like.

**Pull request merged**